test/commons/color/get-text-shadow-colors.js

```diff
@@ -152,4 +168,28 @@ describe('axe.commons.color.getTextShadowColors', function() {
     assert.lengthOf(shadowColors, 1);
     assert.equal(shadowColors[0].blue, 255);
   });
+
+  it('returns a transparent shadow when x offset is greater than blur', function() {
+    fixture.innerHTML =
+      '<span style="text-shadow: ' + '1px 0 0 #F00' + '">Hello world</span>';
```

Comment:
Prettier didn't know what you were on about. So lets keep it at:

```suggestion
      '<span style="text-shadow: 1px 0 0 #F00">Hello world</span>';
```